config/opensearch/service.yml

```diff
@@ -1,4 +1,5 @@
 index_spec: config/opensearch/index-spec.json
 max_num_segments: 10
-index_thread: 5
-query_thread: 2
+index_thread_qty: 1
+bulk_size: 10000
```

Comment:
Could you change default on this to 500?

Reply:
Done